package.json

```diff
@@ -16,10 +16,11 @@
     "bundle": "browserify src/js/main.js > dist/js/script.js",
     "sync": "node src/index.js & browser-sync start --proxy 'localhost:8080' --files 'src'",
     "src:test": "babel --plugins transform-object-rest-spread src/js/pre-logic.js > src/js/logic.js && browserify src/js/main.js > src/js/script.js && npm run sync",
```

Comment:
sure, I agree, I've fixed this - it wasn't working because --presets need to be in brackets [] - e.g. `browserify src/js/main.js -o dist/js/script.js -t [ babelify --presets [ es2015 ] --plugins [transform-object-rest-spread]]`